HeroVault/HeroVault/CoreData/CoreDataHandler.swift

```diff
@@ -31,6 +31,7 @@ protocol CoreDataHandlerType {
     func saveObjectIntoCoreData(_ object: MarvelData)
     func deleteObjectFromCoreData(_ object: MarvelData)
     func deleteAllObjectsFromCoreData()
+    func signUpUser(userName: String, password: String) -> Bool
     //delete this one
```

Comment:
please add the login user to the protocol as well. All public functions sould be in the protocol